### ISSUE TITLE
Can't add a doc comment on pub SomeError

### ISSUE THREAD

**Issue by jamesray1** [lovasoa/custom_error#15]
What I tried:
```rust
custom_error!{
    /// The error type for crate_name.
    pub GError
   // stuff goes here
}
```

I got a compiler error for the doc comment:
```
no rules expected the token `#`

no rules expected this token in macro call(undefined)
errors.rs(11, 5): no rules expected this token in macro call
```

**Comment by lovasoa** [comment]
It looks like this could be implemented using the solution described in https://github.com/rust-lang/rust/issues/37277

**Comment by lovasoa** [comment]
The feature was implemented and works starting with version 1.5.1.

Example:

```rust
custom_error!{
   /// This is the documentation for my error type
   pub MyError A="error a", B="error b"
}
```
